src/utils/netdata-sdk/metrics-stream.ts

```diff
@@ -16,14 +16,16 @@ interface FetchInputEvent {
   onSuccessCallback: (data: { [key: string]: unknown }) => void
 }
 
+const METRICS_TIMEOUT = 15_000
```

Comment:
Strange annotation :) I didn't know about it

Reply:
yeah, it came quite recently into typescript :)